src/lib/template/helpers/pathParamsToDomainParams.ts

```diff
@@ -92,17 +92,8 @@ export default function (method: string, pathObject: any, withType: boolean = fa
 
   // Inject the response to the params
   if (pathObject['x-passResponse'] && tplType !== 'client') {
-    if (fileType === 'STUB') {
-      params.push(
-        'res' + addType(
-        withType,
-        pathObject,
-        undefined,
-        undefined,
-        ));
-    } else {
-      params.push('res' + addType(withType, pathObject));
-    }
+    const type = withType ? ': Response' : '';
```

Comment:
This is a fairly critical function and will need to close attention to testing - not going to have time on this one till later next week i think

Reply:
sure thing - yeah no rush.
was just a minor refactor as the 2 if else conditions passed the same 2 args to the method, and only the first is actually used so its just a slight streamline